packages/@n8n/api-types/src/chat-hub.ts

```diff
@@ -119,6 +120,46 @@ export const emptyChatModelsResponse: ChatModelsResponse = {
 	'custom-agent': { models: [] },
 };
 
+export const binaryFileTypeSchema = z.enum([
+	'text',
+	'json',
+	'image',
+	'audio',
+	'video',
+	'pdf',
+	'html',
+]);
+
+export const binaryDataSchema = z.object({
+	data: z.string(),
+	mimeType: z
```

Comment:
Accepting a user provided mime type as is can be a security risk if we do any processing based on it. What do we do with these? Are they processed in any ways or just stored in there?